SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -94,37 +94,37 @@ BattleCoord = Black
 # Es deaktiviert „den Monat der SEUCHE!“
 # Απενεργοποιεί «τον μήνα της ΠΑΝΟΥΚΛΑΣ!»
 # Wyłącza "miesiąc PLAGI!".
-DisablePlague = 1
+DisablePlague = 0
 
 # Disables the 12 double growth months and the spawning monsters on map
 # NOTE: does *NOT* remove "Month of the Imp"
 # Отключение генерации месяцов двойного прироста и "абстрактного" существа
 # ПРИМЕЧАНИЕ: опция НЕ отключает "Неделю/месяц Беса"
-# Es deaktiviert die 12 Doppeltwachstumsmonate und die laichenden Monster auf der Karte 
+# Es deaktiviert die 12 Doppeltwachstumsmonate und die laichenden Monster auf der Karte
 # ANMERKUNG: Es streicht *NICHT* „den Monat des Imps“
 # Απενεργοποιεί τους 12 μήνες διπλής ανάπτυξης και την εμφάνιση τεράτων στον χάρτη
-# ΣΗΜΕΙΩΣΗ: *ΔΕΝ* αφαιρεί τον «Μήνα του Ιμπ» 
+# ΣΗΜΕΙΩΣΗ: *ΔΕΝ* αφαιρεί τον «Μήνα του Ιμπ»
 # Wyłącza miesiące 12 stworzeń, które dublują swoją populację i pojawiają się na mapie.
 # UWAGA: *NIE* usuwa "Miesiąca Impa".
-DisableDoubleGrowth = 1
+DisableDoubleGrowth = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.